code/script.js

```diff
@@ -1,51 +1,312 @@
+    <ul class="bot-typing">
+      <li></li>
+      <li></li>
+      <li></li>
+    </ul> 
+  </div>
+
+  ` 
+}
+
+// Removes bot typing effect
+
+const removeTyping = () => {
+  document.getElementById('preloader').style.display='none'
+}
 
-// Declare your functions after this comment
 
 // This function will add a chat bubble in the correct place based on who the sender is
+
 const showMessage = (message, sender) => {
-  // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
+  // the if statement checks if the sender is 'user' and if that's the case it inserts an html section inside the chat with the posted message
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
-          <p>${message}</p>
+          <p class="p-bubble-msg">${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/bot-guest.svg" alt="User" />  
       </section>
     `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
+    // the else if statement checks if the sender is a bot and if that's the case it inserts an html section inside the chat with the posted message
   } else if (sender === 'bot') {
+    //console.log('bot-message') //added console log
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
+        <img src="assets/bot-hotel.svg" alt="Bot" />
+            <div class="bubble bot-bubble">
+              <p class="p-bubble-msg">${message}</p>
         </div>
       </section>
     `
-  }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  } 
+    // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box and adds sound
   chat.scrollTop = chat.scrollHeight;
+  messageSound();
 }
 
-// Starts here
+
+// Conversations starts here
+
+// Question 1 - Bot asks 'What is your room number'?
+
 const greeting = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot');
-  // Just to check it out, change 'bot' to 'user' here 👆
+      showMessage("Welcome to Hotel Vosje. What is your room number?", 'bot'),2000; 
+    };
+  
+
+// User answers with room number
+
+const handleFormInput = (event) => {        // at submit this function will be invoked
+  event.preventDefault();        // prevents website refresh at submit
+  const room = roomInput.value;        // input value will be stored in the const name
```

Comment:
Could it be a good idea to block that the bot takes strings, so that it only takes numbers? Right now you can write your name instead of room number and it still works.

Reply:
good idea and good spotting. we actually talked about that but decided to leave it like is because maybe hotels have names or letters in the rooms combined. like "top suite 3" or "panther room" (lol) or something. :)